### ISSUE TITLE
I was able to log in without password. Is this really 2FA?

### ISSUE THREAD

**Issue by jensb** [nextcloud/twofactor_webauthn#296]
### Question

I am using the external_user app on Nextcloud 24 to log users in using their IMAP credentials, which works fine.
When I add WebAuthn as a 2FA option, users can log into their Nextcloud accounts just by providing their login name, without password.
I would like to offer users the WebAuthn **in addition to** the usual login/password prompt, i.e. as a real second factor.
If I understand correctly the (deprecated) U2F app allowed this, and WebAuthn is supposed to replace U2F.
So is this possible using this app?

**Comment by ChristophWurst** [comment]
You are using passwordless authentication, aren't you? Those are two concepts. Both use the webauthn technology.

This app does not disable password auth.

**Comment by jensb** [comment]
No, but - as I see it - it offers only to **replace** password auth, not **add** to it. Correct?

I would like to be able to configure my account to require login and password **and a WebAuthn device**.

I this possible?

**Comment by ChristophWurst** [comment]
Could you please share a screenshot of your login page?
